scripts/zowe-install-MVS.sh

```diff
@@ -133,6 +133,22 @@ else
     echo "  $script failed to create ${ZOWE_DSN_PREFIX}.SZWESAMP, RC=$rc" >> $LOG_FILE
 fi
 
+# 3. {datasetprefix}.SZWEPLUG
+
+tsocmd "delete '${ZOWE_DSN_PREFIX}.SZWESAMP' " >> ${LOG_FILE} 2>&1
```

Comment:
why is a ticket labeled create doing a delete?
i can't think of a reason this is here so please explain.